prefs/Resources/fr.lproj/Common.strings

```diff
@@ -18,7 +18,7 @@
 
 	<!-- Heading above the "Email Support" button. -->
 	<key>EMAIL_SUPPORT_HEADER</key>
-	<string>Des problèmes?</string>
+	<string>Des problèmes ?</string>
```

Comment:
I'm almost sure French people don't write a space here, not from what I've learned.